meson.build

```diff
@@ -378,25 +378,51 @@ install_data(pgroonga_database_update_sqls,
   install_dir: pg_sharedir / 'extension',
 )
 
-pg_regress = pg_pkglibdir / 'pgxs/src/test/regress/pg_regress'
-run_command('mkdir', '-p', meson.current_build_dir() / 'results/function/highlight-html/', check: true)
-run_command('mkdir', '-p', meson.current_build_dir() / 'results/function/command/', check: true)
+ruby = find_program('ruby')
+pg_regress = find_program(
+  'pg_regress',
+  dirs: [pg_pkglibdir / 'pgxs/src/test/regress']
+)
 
-test('regression',
-  find_program(pg_regress),
-  args: [
-    '--inputdir=' + meson.current_source_dir(),
-    '--outputdir=' + meson.current_build_dir(),
-    '--bindir=' + pg_bindir,
-    '--dlpath=' + meson.current_build_dir(),
-    '--load-extension=pgroonga',
-    'function/highlight-html/multiple-keywords',
-    'function/command/select'
+schedule = meson.current_build_dir() / 'schedule'
+prepare_test = custom_target(
+  'prepare',
+  output: 'schedule',
+  command: [
+    ruby,
+    meson.current_source_dir() / 'test' / 'prepare.rb',
+    schedule
   ],
-  workdir: meson.current_source_dir(),
-  depends: [pgroonga],
-  suite: 'regression',
   env: {
-    'PG_REGRESS_DIFF_OPTS': '-u --color=always',
-  },
+    'BUILD_DIR': meson.current_build_dir(),
+  }
+)
+
+test('regression',
+     pg_regress,
+     args: [
+       '--bindir', pg_bindir,
+       '--dlpath', meson.current_build_dir(),
+       '--inputdir', meson.current_source_dir(),
+       '--outputdir', meson.current_build_dir(),
+       '--launcher=' + meson.current_source_dir() / 'test' / 'short-pgappname',
+       '--load-extension=pgroonga',
+       f'--schedule=@schedule@'
```

Comment:
Can we avoid mixing `--XXX=ZZZ` and `--XXX ZZZ`?

Reply:
fix: [9b30e57](https://github.com/pgroonga/pgroonga/pull/753/commits/9b30e57103182c8300d6225ca9ac6284405260d4)